docker-compose.yml

```diff
@@ -2,13 +2,6 @@ version: '3.5'
 
 services:
   backend:
-    build:
-      context: .
-      dockerfile: ./docker/images/backend/Dockerfile
-      target: moodlenet
-      args:
-        BACKEND_VERSION: "master"
-        DOCKER_USER: ${DOCKER_USER:-1000}
     user: ${DOCKER_USER:-1000}
```

Comment:
So we are forced to build the image using the `bin/build` script? I would prefer not having this kind of dependency and be able to build using docker-compose descriptive format with appropriate defaults.

Reply:
After our talk, I restored the build in `docker-compose.yml` with @lunika 's original suggestion for the `BACKEND_VERSION` default value.

And the `bin/build` script is now using `docker-compose` instead of `docker` to build the image.